AppCenter/AppCenterTests/MSDBStorageTests.m

```diff
@@ -385,7 +385,7 @@ - (void)testDroppAllTables {
   XCTAssertTrue([self tableExists:tableName1]);
   XCTAssertTrue([self.sut createTable:tableName2 columnsSchema:self.schema[kMSTestTableName]]);
   XCTAssertTrue([self tableExists:tableName2]);
-  [self.sut dropAllTables];
+  [self.sut dropDatabase];
 
   // Then
   XCTAssertFalse([self tableExists:tableName1]);
```

Comment:
The assertions here should change to verify that the database no longer exists

Reply:
I keep this one and add file check assertion.